lib/capybara/node/matchers.rb

```diff
@@ -322,6 +322,31 @@ def has_no_css?(path, **options, &optional_filter_block)
         has_no_selector?(:css, path, **options, &optional_filter_block)
       end
 
+      ##
+      #
+      # Checks if the page or current node has a element with the given
+      # local name.
+      #
+      # @param [String] locator           The local name of a element to check for
+      # @option options [String, Regexp] :href    The value the href attribute must be
```

Comment:
There is no special href attribute support

Reply:
Whoops! Thank you.